qemu/tests/cfg/multi_disk_random_hotplug.cfg

```diff
@@ -31,6 +31,9 @@
                 stg_params = "fmt:virtio,virtio_scsi,usb2"
             usbs += " ehci"
             usb_type_ehci = usb-ehci
+            Host_RHEL.m6:
+                usbs= "ehci"
```

Comment:
@xutian That's for the bz, i.e. fail to attach the usb-tablet to usb-ehci controller on RHEL6, so turn to use the original ich9-usb-ehci1/ich9-usb-uhci* controllers; I don't think we must use multi usb controllers as that in RHEL7 in the case scenario. How do you think?